gf-webapp-admin/src/component/rooms/room/Room.module.scss

```diff
@@ -14,6 +14,18 @@
     margin-bottom: 5px;
     cursor: pointer;
   }
+
+  button {
+    display: flex;
+    align-items: center;
+    width: auto;
+    height: auto;
+    font-size: 1rem;
+    animation: none;
+    font-family: 'Lato Regular', sans-serif, 'Arial';
```

Comment:
Check all files where we don't use familu Lato. If everywhere Lato? then better write it with * in index.scss